requirements/pip.txt

```diff
@@ -4,11 +4,11 @@
 #
 #    make upgrade
 #
-wheel==0.37.1
+wheel==0.38.2
```

Comment:
Dropped support for `Python<3.7` in new version: https://wheel.readthedocs.io/en/stable/news.html